navigation_and_routing/lib/src/screens/authors.dart

```diff
@@ -4,25 +4,27 @@
 
 import 'package:flutter/material.dart';
 
+import '../data/author.dart';
 import '../data/library.dart';
-import '../routing.dart';
 import '../widgets/author_list.dart';
 
 class AuthorsScreen extends StatelessWidget {
   final String title = 'Authors';
+  final ValueChanged<Author> onTap;
 
-  const AuthorsScreen({super.key});
+  const AuthorsScreen({
+    required this.onTap,
+    super.key,
+  });
 
   @override
   Widget build(BuildContext context) => Scaffold(
         appBar: AppBar(
-          title: Text(title),
+          title: const Text(title),
```

Comment:
I'm not sure why the analyzer isn't complaining here on newer SDK builds, but this can't be const.

```suggestion
          // ignore: prefer_const_constructors
          title: Text(title),
```

I'll look for an existing SDK issue or open one.

Reply:
Ah you fixed it already, thanks!

Reply:
The `title` field is not const, so this makes sense. I made it a `final` field and assigned the value in the constructor so that the `Text` widget doesn't need to be const. Not the most sensible fix, but the easiest for the different versions of the analyzer to understand 🤷

Reply:
Sounds good to me. Could also make it `static const` if you don't want the title to be changed.

I filed https://github.com/dart-lang/sdk/issues/53927